api_spec.yaml

```
@@ -8124,6 +8124,318 @@ paths:
         '500':
           $ref: '#/components/responses/500'
 
+  /appid-<appId>/<tenantId>/recipe/saml/clients/create:
```

If we are using PUT then we should have it as saml/clients, if we are doing create then it should be POST (and/or rename it to `POST createOrUpdate`)
```suggestion
  /appid-<appId>/<tenantId>/recipe/saml/clients:
```